packages/components/src/components/Log/Log.js

```diff
@@ -25,21 +25,44 @@ const LogLine = ({ data, index, style }) => (
   </div>
 );
 
-class Log extends Component {
-  getLogList() {
-    const { status, intl } = this.props;
+const trailers = {
+  Completed: {
```

Comment:
not sure if this will work (not using intl.formatMessage here, and passing these as variables later) when we add tooling to extract the strings automatically, but I'll keep an eye on it